### PR TITLE
added SoundMuffler to addon page

### PR DESCRIPTION
**Pull request by J3fftw1**
added my SoundMuffler to the addon page

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Name | Author(s) | Description | Info | GitHub | Download
...
 SlimeScrap | John000708 | <details><summary>Click to expand</summary>This addon allows you to break down machines and other items into their components - useful for recycling old items.</details> | [BukkitDev](https://dev.bukkit.org/projects/slimescrap) | N/A | [BukkitDev](https://dev.bukkit.org/projects/slimescrap/files)
 SlimeCalculator | John000708 | <details><summary>Click to expand</summary>This Addon gives you the ability to calculate the resources needed to craft Slimefun items.</details> | [BukkitDev](https://dev.bukkit.org/projects/slimecalculator) | N/A | [BukkitDev](https://dev.bukkit.org/projects/slimecalculator/files)
+SoundMuffler | J3fftw1 | <details><summary>Click to expand</summary>This Addon adds machines that have the abbility to muffle sound.</details> | N/A | [Github](https://github.com/J3fftw1/SoundMuffler) | N/A
```
Comment (WalshyDev) [inline pages/Addons.md]: *adds a machine which has the ability to muffle sound.

Comment (WalshyDev) [reply pages/Addons.md]: Also, you should link straight to the releases page rather than the main repo.

Comment (TheBusyBiscuit) [reply pages/Addons.md]: It should link to the main repo and not the releases page...
It's listed as "SOURCE CODE" Walshy.

Comment (WalshyDev) [reply pages/Addons.md]: Oh I thought he did that under the download part LUL

Well, the download part should link right to releases

**Pull request closed without merging**